fdbserver/LogSystem.h

```diff
@@ -1104,6 +1110,7 @@ struct LogPushData : NonCopyable {
 	Reference<ILogSystem> logSystem;
 	std::vector<Tag> next_message_tags;
 	std::vector<Tag> prev_tags;
+	std::vector<Tag> written_tags;
```

Comment:
This can be a `set` so that duplicated tags are avoided.

Reply:
Done.